modules/backend/classes/NavigationManager.php

```diff
@@ -195,6 +199,26 @@ public function registerMenuItems($owner, array $definitions)
             $this->items = [];
         }
 
+        $validator = Validator::make($definitions, [
+            '*.label' => 'required',
+            '*.icon' => 'required_without:*.iconSvg',
+            '*.iconSvg' => 'required_without:*.icon',
```

Comment:
@Samuell1 You can drop this validation rule - the `*.icon` rule covers both cases.